scripts/create_incr_tickets.py

```diff
@@ -0,0 +1,169 @@
+import os
+import sys
+from collections import namedtuple
+
+# We should aspire to create batches of less than 15 python files
+# although this is not a strict limit
+MAX_FILE_NUMBER = 10
```

Comment:
Comment says 15 but var is set to 10?

Reply:
Yeah, it's a little hard to express. I suppose having `MAX` in the name doesn't help. It's more of a target value. I want to average 15, so setting it to 10 means that sometimes we get just 10 in a batch, and occasionally we get a large batch that exceeds 15. Setting it to 10 really reduces the occurrence of large batches. How would you suggest I clean this up?

Reply:
`TARGET_BATCH_SIZE`?